.azure-ci.yml

```diff
@@ -8,8 +8,13 @@ trigger:
     include:
     - master
     - release/3.0
+  paths:
+    exclude:
+      - Documentation/*
+      - README.md
```

Comment:
Can't you just do: `*.md`?

Should this paths be on the `pr` triggers section as well? I think they do.

Reply:
good catch, will do that.